docs/getting-started/devenv/using-visual-studio.md

```diff
@@ -1,13 +1,229 @@
-# 🔧 Development using Visual Studio 2015
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/281) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# Development using Visual Studio 2015
+
+by [Bertrand Le Roy](https://github.com/bleroy) and [Phillip Carter](https://github.com/cartermp)
+
+Visual Studio 2015 provides a full-featured development environment for developing .NET Core applications. The scripts in this document describe the steps necessary to build a number of typical .NET Core solutions, or solutions that include .NET Core components, using Visual Studio. The scenarios include testing and using third-party libraries that have not been explicitly built for the most recent version of .NET Core.
+
+## Prerequisites
+
+* Visual Studio 2015 Update 2. If you don't have Visual Studio already, you can download [Visual Studio Community 2015](https://www.visualstudio.com/products/visual-studio-community-vs) for free. 
+
+* NuGet Manager extension for Visual Studio. NuGet is the package manager for the Microsoft development platform including .NET Core. When you use NuGet to install a package, it copies the library files to your solution and automatically updates your project (add references, change config files, etc.). 
+
+* .NET Core Tooling Preview 1 for Visual Studio 2015. This installs templates and other tools as well as .NET Core itself.
+
+* A supported version of the Windows client or server operating system. For a list of supported versions, see [.NET Core System Requirements](../system-reqs.md). 
+
+## Getting Started
+
+The following steps will set up Visual Studio 2015 for .NET Core development:
+
+1. Verify that you're running Visual Studio 2015 Update 2:
+
+   * On the **File** menu, choose **Help**, **About Microsoft Visual Studio**.
+
+   * In the About Microsoft Visual Studio dialog, the version number should include "Update 2" (or higher).
+
+2. Download and install the [Visual Studio official MSI Installer](https://go.microsoft.com/fwlink/?linkid=798481). This will install the .NET Core Tooling Preview 1 for Visual Studio 2015.
```

Comment:
"Visual Studio official MSI Installer" sounds like it will install VS itself. Maybe:

```md
Download and install the [official MSI Installer of .NET Core Tooling Preview 1 for Visual Studio 2015](https://go.microsoft.com/fwlink/?linkid=798481).
```